### DISCUSSION TITLE
Certificate creation stopped working

### DISCUSSION THREAD

**Discussion by trendmonkey** [NginxProxyManager/nginx-proxy-manager#3562] [Q&A]
Hi,

My NPM installation on docker was working totally fine when I last used it a week ago. When I go to create a certificate today I get the following error. Has anyone seen this or have any idea how to fix it? I'm a bit worried none of my certs will renew when the time comes.

> CommandError: nginx: [emerg] cannot load certificate "/etc/letsencrypt/live/npm-6/fullchain.pem": BIO_new_file() failed (SSL: error:80000002:system library::No such file or directory:calling fopen(/etc/letsencrypt/live/npm-6/fullchain.pem, r) error:10000080:BIO routines::no such file)

> nginx: configuration file /etc/nginx/nginx.conf test failed
> 
>     at /app/lib/utils.js:16:13
>     at ChildProcess.exithandler (node:child_process:430:5)
>     at ChildProcess.emit (node:events:518:28)
>     at maybeClose (node:internal/child_process:1105:16)
>     at ChildProcess._handle.onexit (node:internal/child_process:305:5)

Many thanks for any help!
Cheers, Greg

**Comment by trendmonkey** [comment]
OK, I've discovered that this happens if a certificate is deleted but there is still a proxy host that is pointing to it.
Just delete or disable the proxy host and all is well.

**Reply by Hutch79** [reply]
How did you delete the host?
I can't even access the UI.
